website/config/environment.js

```diff
@@ -38,8 +41,8 @@ module.exports = function (environment) {
   if (environment === 'development') {
     // ENV.APP.LOG_RESOLVER = true;
     // ENV.APP.LOG_ACTIVE_GENERATION = true;
-    // ENV.APP.LOG_TRANSITIONS = true;
-    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
+    ENV.APP.LOG_TRANSITIONS = true;
+    ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
```

Comment:
do we need these uncommented? there is  already a lot of noise in the console when building the app, and we may risk to lose important messages there

Reply:
@MelSumner can we get these commented out again so we can get this PR finished?